packages/wrangler/src/dev/miniflare.ts

```diff
@@ -728,6 +728,9 @@ export function handleRuntimeStdio(stdout: Readable, stderr: Readable) {
 		isCodeMovedWarning(chunk: string) {
 			return /CODE_MOVED for unknown code block/.test(chunk);
 		},
+		isAccessViolation(chunk: string) {
+			return chunk.includes("access violation;");
```

Comment:
I think we should be a bit more specific here

```suggestion
			return /Received structured exception .+? access violation;/.test(chunk);
```

Reply:
I thought about making it more strict but I am more concerned about false negatives than false positives in this case. And making it too strict might mean that we miss this going forward if the messaging changes slightly.
This check is already gated on there being a stack trace which implies a catastrophic workerd error already.